python/selfie-lib/selfie_lib/ArrayMap.py

```diff
@@ -33,32 +41,41 @@ def __getitem__(self, index: Union[int, slice]) -> Union[K, List[K]]:
             return self.__data[index]
         else:
             raise TypeError("Invalid argument type.")
+        
+    def plusOrThis(self, element: K) -> 'ArraySet[K]':
+        if element not in self.__data:
+            self.__data.append(element)
```

Comment:
You don't want to mutate the existing structure, and you want to make sure that `ArraySet` stays sorted.

https://github.com/diffplug/selfie-python-wip/blob/c41e68cb1141e2f7b6902651b4b3c45632283a07/jvm/selfie-lib/src/commonMain/kotlin/com/diffplug/selfie/ArrayMap.kt#L257-L284